constance/backends/database/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
+
+
+class ConstanceConfig(AppConfig):
```

Comment:
Ah, one other thing, please rename the app config class to `ConstanceDatabaseConfig` to not confuse it with the other instances of "config" elsewhere in the app